lib/ansible/module_utils/facts/virtual/linux.py

```diff
@@ -139,17 +139,24 @@ def get_virtual_facts(self):
             virtual_facts['virtualization_role'] = 'guest'
 
         product_name = get_file_content('/sys/devices/virtual/dmi/id/product_name')
+        product_family = get_file_content('/sys/devices/virtual/dmi/id/product_family')
 
         if product_name in ('KVM', 'KVM Server', 'Bochs', 'AHV'):
             guest_tech.add('kvm')
             if not found_virt:
                 virtual_facts['virtualization_type'] = 'kvm'
                 found_virt = True
 
-        if product_name == 'RHEV Hypervisor':
-            guest_tech.add('RHEV')
+        if product_name == 'oVirt Node':
```

Comment:
Hmm, that's only for node hosts, but shouldn't we add below also to cover CentOS hosts?

```
if product_name == 'oVirt Node' or sys_vendor == 'oVirt':
...
```